magento/magento2
# Add custom attribute values to the origData property to prevent unnecessary db inserts and updates when saving the customer entity

### Description (*)
Prevents the creation and execution of unnecessary db inserts/updates when saving a customer entity when non-static customer custom attribute exist.

### Fixed Issues (if relevant)
1. Fixes magento/magento2#38753

### Manual testing scenarios (*)
1. Create a non-static customer custom attribute (e.g. `my_custom_attribute`)
2. Set `my_custom_attribute` to any value and save
3. Enable DB profiling
4. Load up the customer using the Customer Repository (`\Magento\Customer\Api\CustomerRepositoryInterface`)
5. Without changing any value save the customer using the Customer Repository (`\Magento\Customer\Api\CustomerRepositoryInterface::save()`)
6. Check DB profiling results
7. Notice there is no longer a query to insert the value that was already saved and set for your custom attribute `my_custom_attribute` since the original value does not differ from the new value


### Contribution checklist (*)
